floodscan_susceptibliity_target_pipeline.R

```diff
@@ -223,6 +265,19 @@ list(
       ) %>%
       mutate(pct_exposed = mean_exposed / total_pop)
   ),
+  tar_target(
+    name = df_eth_ssn_adm1_mean_stat,
+    df_eth_ssn_adm1_stats_bin_method %>%
+      mutate(season = case_when(month(date) %in% c(4) ~ "AMJ",
+                                .default = "")) %>%
```

Comment:
you may want to set `.default=NA_character_` and then filter it out before making the `{gt}` tables? currently it's left in, but not clearly marked in the tables as to what it is